app/src/main/res/xml/preferences.xml

```diff
@@ -70,6 +70,12 @@
           app:singleLineTitle="false"
           android:summary="@string/display_campaigns_explanation"
           android:title="@string/display_campaigns" />
+
+        <SwitchPreference
+            android:defaultValue="false"
```

Comment:
I still prefer keeping this turned on by default (ignoring the inversion mentioned above) with a way to suggest turning it off in a particular case. Given this has merged, I'll open a new issue about it with some details.

Reply:
I would disagree on this one, priority number one is to avoid any data loss, this is far more important than any potential convenience.

Reply:
My suggestion tries to balance both the data loss and convenience 🙂 

More details in this issue: #5242